README.md

```diff
@@ -25,12 +29,13 @@ Responsibilities and roles of the bridge:
   - set minimum required signatures from validators in order to relay a user's transaction
 - Validator Role :
   - provide 100% uptime to relay transactions
-  - listen for Deposit events on Home bridge to mint erc20 token on Foreign bridge
-  - listen for Withdraw events on Foreign bridge to unlock funds on Home Bridge
+  - listen for `UserRequestForSignature` events on Home Bridge and sign an approval to relay assets on Foreign network;
+  - listen for `CollectedSignatures` events on Home Bridge as soon enough signatures are collected and transfer all collected signatures to Foreign Bridge contact;
+  - listen for `UserRequestForAffirmation` or `Transfer` (depending on the bridge mode) events on Foreign Bridge and send approval to Home Bridge to relay assets from Foreign Network to Home
 - User role:
-  - sends POA coins to Home bridge in order to receive ERC20 token on Foreign Bridge
-  - sends ERC20 POA20 token on Foreign Bridge in order to receive POA coins on Home Bridge
-
+  - sends assets to Bridge contracts:
+    - in `NATIVE-TO-ERC` mode: send native coins to Home Bridge to receive ERC20 tokens from Foreign Bridge, send ERC20 tokens to Foreign Bridge to unlock native coins from Home Bride;
+    - in `ERC-TO-ERC` mode: transfer ERC20 tokens to Foreign Bridge to mint ERC20 tokens on Home Network, transfer ERC20 tokens to Home Bridge to unlock ERC20 tokens on Foreing networks. 
```

Comment:
- in `ERC-TO-ERC` mode: transfer ERC20 tokens to **the** Foreign Bridge to mint ERC20 tokens on **the** Home Network, transfer ERC20 tokens to **the** Home Bridge to unlock ERC20 tokens on **Foreign** networks.